docs/import_export/README.md

```diff
@@ -0,0 +1,68 @@
+# Import/Export Documentation for uPortal 5
+
+## Overview
+The Import/Export tool in uPortal 5 allows users to transfer configuration data and settings between instances, facilitating data migration and backup.
+
+## Supported Operations
+- **IMPORT**: Adds or updates entities from XML files in the portal database.
+- **LIST**: Lists supported data types and existing entities.
+- **EXPORT**: Creates XML files of specified entities and saves them to a location.
+- **DELETE**: Removes specified entities and their dependencies from the portal database.
+
+## Importing Data
+
+### Import Multiple Files
+```bash
+ant data-import -Ddir={path-to-directory} [-Dpattern={ant-pattern}]
+```
+
+### Import Single File
+```bash
+ant data-import -Dfile={path-to-file}
+```
+
+### Import Single File (uPortal 4.3+)
+```bash
+ant data-import -Dfiles={comma-separated-list-of-files}
+```
+
+### Import List File (uPortal 4.3+)
+```bash
+ant data-import -DfilesListFile={path-to-file}
+```
+
+## Listing Data
+
+### List Types
+```bash
+ant data-list
+```
+
+### List Data of a Specific Type
+```bash
+ant data-list -Dtype={entity-type}
+```
+
```

Comment:
````suggestion
ant data-list -Dtype={entity-type}
```

Replace `{entity-type}` with the type of entity you wish to list (e.g., `portlets`, `groups`).
````